.github/workflows/Build.yaml

```
@@ -0,0 +1,146 @@
+                    exit 1
+                fi
+
+            - name: Download dependencies
+              run: go mod download
+
+            - name: Run tests
+              run: |
+                  make test
+                  if [[ ! -z $(git status -s) ]]
+                  then
+                    echo "go test - errors running go tests : $(git status -s)"
+                    exit 1
+                  fi
+
+            - name: Set up Python 3.x on PR branch Part 1
+              uses: actions/setup-python@v2
+              with:
+                python-version: '3.9'
+
+            - name: Set up Python 3.x on PR branch Part 2
+              run: |
+                # set up python requirements and scripts on pr branch
+                python3 -m venv ve1
+                cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+                cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+            - name: Check if only release file in PR
+              id: check_version_in_PR
+              run: |
+                # check if release file only is included in PR
+                ve1/bin/release-checker --api-url=${{ github.event.pull_request._links.self.href }}
+
+            - name: Build and Test Image
+              id: build_and_test
+              run: |
+                # build and test a docker image
+                ve1/bin/build-and-test --image-name=${{steps.check_version_in_PR.outputs.PR_release_image}} --sha-value=${GITHUB_SHA} --verifier-version=${{steps.check_version_in_PR.outputs.PR_version}}
+
+            - name: Checkout main
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              uses: actions/checkout@v2
+              with:
+                ref: main
+
+            - name: Set up Python 3.x on main branch Part 1
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              uses: actions/setup-python@v2
+              with:
+                python-version: '3.9'
+
+            - name: Set up Python 3.x on main branch Part 2
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              run: |
+                # set up python requirements and scripts on main branch
+                python3 -m venv ve1
+                cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+                cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+            - name: Check for restricted files and user permissiom
+              id: check_authorization
+              run: |
+                # check for a restricted file and, if found, check user has permissiom
+                ve1/bin/check-user --api-url=${{ github.event.pull_request._links.self.href }} --user=${{ github.event.pull_request.user.login }}
+
+            - name: Check if version updated
+              id: check-version_updated
```

```suggestion
              id: check_version_updated
```

To make the style consistent.

---

Updated

---

I'm still seeing the unchanged version...